README.md

```diff
@@ -352,6 +373,31 @@ to reconnect to the instance with the new IP.
 You can also store this in a role as an attribute or assign to the
 node directly, if preferred.
 
+## aws_elastic_network_interface
+
+The `elastic_network_interface` resource provider does not support creating new
+network interfaces. This must be done before running a recipe that uses the resource.
+After creating a new Elastic Network Interface, we recommend storing it in a
+databag and loading the item in the recipe.
```

Comment:
This doesn't reflect the reality of what the `:create` action actually does; needs updating.